website/content/en/docs/concepts/nodepools.md

```diff
@@ -229,6 +261,12 @@ Karpenter can be configured to create nodes in a particular zone. Note that the
 [Learn more about Availability Zone
 IDs.](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
 
+**Static NodePool**
+
+Topology requirement field is the source of truth for topology decisions. Users who want to spread nodes across zones can do so explicitly by:
+- Specifying multiple zones in the topology.kubernetes.io/zone requirement, or
```

Comment:
We didn't add implicit zonal spread for the initial release, right? The only way to guarantee zonal spread at the moment is to create multiple static NodePools. Regardless, I think we can drop these details from here, guidance for how to configure static NodePools to achieve certain requirements can be deferred for a "tasks" page. If we feel strongly about including it now, I'd add this in the examples section.